tests/main.test.ts

```diff
@@ -25,7 +25,7 @@ describe("Plugin tests", () => {
   });
 
   it("Should serve the manifest file", async () => {
-    const worker = require("../src/worker").default;
+    const { default: worker } = await import("../src/worker");
```

Comment:
Default is a hack?

Reply:
What do you mean? Default is the default exported from the file. Like export default xxx